frontend/packages/client/src/contexts/NotificationService.js

```diff
@@ -88,13 +87,15 @@ const NotificationServiceProvider = ({ children }) => {
   const subscribeCommunity = async (communityId) => {
     try {
       //here we call api
+      const apiResponse = true;
       setNotificationSettings((prevState) => ({
         ...prevState,
-        communitySubscription: {
+        communitySubscription: [
           ...prevState.communitySubscription,
-          [communityId]: true,
-        },
+          communityId,
+        ],
       }));
+      return apiResponse;
```

Comment:
these were being turned into objects but they should be arrays so I re-wrote them to use arrays and added a fake api response for testing purposes